### DISCUSSION TITLE
status-symbols should be colored

### DISCUSSION THREAD

**Discussion by karba71** [Gulivertx/cbatmo#46] [General]
**Hello**

I think the status-symbols should be colored (similar netatmo pc-app).
Actually it's very difficult to distinguish.

Status-Symbole 20 11 20
<img width="379" alt="Status-Symbole 20 11 20" src="https://user-images.githubusercontent.com/56199098/101227283-93a5a480-3697-11eb-8be6-1183fb0d5251.PNG">

Karba

**Comment by Gulivertx** [comment]
Hi,
Sounds like good idea, where did you find the graphic resources?

**Comment by karba71** [comment]
Hello,

this pictures are from my netatmo pc-app.
![battery_low](https://user-images.githubusercontent.com/56199098/101241182-44da2800-36f4-11eb-9d3c-836a8bceb12c.png)
![battery_medium](https://user-images.githubusercontent.com/56199098/101241184-49064580-36f4-11eb-8265-40d23f170e91.png)
![battery_high](https://user-images.githubusercontent.com/56199098/101241186-502d5380-36f4-11eb-8e21-62b3e8622e58.png)
![module_int](https://user-images.githubusercontent.com/56199098/101241190-5d4a4280-36f4-11eb-9aac-ec7410424f3b.png)
![module_ext](https://user-images.githubusercontent.com/56199098/101241194-61766000-36f4-11eb-8b61-f3f1c90cdcc3.png)
![module_wind](https://user-images.githubusercontent.com/56199098/101241201-663b1400-36f4-11eb-8df2-1367c2de5803.png)
![module_rain](https://user-images.githubusercontent.com/56199098/101241204-6a673180-36f4-11eb-8e72-92af5ccbe9ab.png)
![signal_low](https://user-images.githubusercontent.com/56199098/101241209-7521c680-36f4-11eb-8c83-a33afd9c7a1a.png)
![signal_high](https://user-images.githubusercontent.com/56199098/101241212-79e67a80-36f4-11eb-9594-493b64c09ba5.png)
![signal_full](https://user-images.githubusercontent.com/56199098/101241220-7eab2e80-36f4-11eb-8a31-56f485262a34.png)
![wifi_full](https://user-images.githubusercontent.com/56199098/101241235-89fe5a00-36f4-11eb-9cdb-80bd9263be99.png)

You mean this symbols?
Some symbols are missing, i will look for this.

**Comment by karba71** [comment]
![signal_medium](https://user-images.githubusercontent.com/56199098/101243134-73f59700-36fe-11eb-803b-16279cc01384.png)

**Comment by karba71** [comment]
![signal_verylow](https://user-images.githubusercontent.com/56199098/101280223-3fe6a880-37c8-11eb-8bd3-5dff100b6869.png)

**Comment by karba71** [comment]
![battery_full](https://user-images.githubusercontent.com/56199098/101280461-78d34d00-37c9-11eb-8a45-57fae0325cf3.png)

**Comment by karba71** [comment]
![battery_unknown](https://user-images.githubusercontent.com/56199098/101283828-44698c00-37dd-11eb-9a6f-fdef3bfaf5d3.png)

![signal_unknown](https://user-images.githubusercontent.com/56199098/101283833-4b909a00-37dd-11eb-9f4f-4cdbd8a3f8d1.png)

**Comment by karba71** [comment]
![wifi_high](https://user-images.githubusercontent.com/56199098/101996391-41393900-3cd2-11eb-8b70-fbae65cde020.png)
![wifi_medium](https://user-images.githubusercontent.com/56199098/101996393-41d1cf80-3cd2-11eb-91a7-411fc78eb2dd.png)
![wifi_low](https://user-images.githubusercontent.com/56199098/101996394-41d1cf80-3cd2-11eb-8560-43fab6411805.png)
![battery_verylow](https://user-images.githubusercontent.com/56199098/101996395-41d1cf80-3cd2-11eb-9117-0d353c0d47b4.png)

**Comment by karba71** [comment]
Hello,

I think this are all symbols.

Karba

**Comment by Gulivertx** [comment]
hmm, I think I cannot use this graphic resources, as they are propriety of Netatmo. I will look after the official developer documents if there is some information about. Anyway, thanks for the image.

**Reply by uwewiest** [reply]
Guliver if you are ok we will create some for you I will attach them here so you can use them and transfer any rights to you

**Comment by Gulivertx** [comment]
I could simply apply a color to the current icon according to the status. But I need to try if the render is correct and if I would like it.

**Reply by uwewiest** [reply]
Yes I did have a look in yout code you "fill" them accordingly I will hurry up with the icons and they will have the same steps (30,50,70 etc)

**Comment by Gulivertx** [comment]
Now integrated in version 2.3.6
<img width="816" alt="Screenshot 2021-01-04 at 21 23 00" src="https://user-images.githubusercontent.com/7717886/103577269-c3cda680-4ed4-11eb-901d-2f8e905a1e54.png">

**Comment by uwewiest** [comment]
Hi here you go with icons please feel free to use them ...100% no copyright my son just made them 

[netatmo_icons_svg.zip](https://github.com/Gulivertx/cbatmo/files/5766798/netatmo_icons_svg.zip)
![signal-2-bars](https://user-images.githubusercontent.com/25363706/103580112-e4e4c600-4ed9-11eb-9740-cbcbd119c659.png)
![signal-3-bars](https://user-images.githubusercontent.com/25363706/103580113-e57d5c80-4ed9-11eb-8ba0-e3d9f4a72c2b.png)
![signal-4-bars](https://user-images.githubusercontent.com/25363706/103580114-e615f300-4ed9-11eb-8311-ce1400ebe391.png)
![signal-5-bars](https://user-images.githubusercontent.com/25363706/103580115-e615f300-4ed9-11eb-8873-9e24f345c4eb.png)
![wifi-1-bar](https://user-images.githubusercontent.com/25363706/103580116-e615f300-4ed9-11eb-9c87-346d121d0f14.png)
![wifi-2-bars](https://user-images.githubusercontent.com/25363706/103580117-e6ae8980-4ed9-11eb-8f1e-257733142fbb.png)
![wifi-3-bars](https://user-images.githubusercontent.com/25363706/103580118-e6ae8980-4ed9-11eb-8054-1799bcaef2c4.png)
![wifi-4-bars](https://user-images.githubusercontent.com/25363706/103580119-e6ae8980-4ed9-11eb-8dfd-93a5339fb66c.png)
![bat_10](https://user-images.githubusercontent.com/25363706/103580122-e7472000-4ed9-11eb-95d3-2597ecf8a13b.png)
![bat_20](https://user-images.githubusercontent.com/25363706/103580124-e7dfb680-4ed9-11eb-86cb-b8daf8834649.png)
![bat_30](https://user-images.githubusercontent.com/25363706/103580125-e7dfb680-4ed9-11eb-8f6c-2ec725c5b880.png)
![bat_40](https://user-images.githubusercontent.com/25363706/103580126-e8784d00-4ed9-11eb-865c-6685292eee5e.png)
![bat_50](https://user-images.githubusercontent.com/25363706/103580127-e8784d00-4ed9-11eb-9661-68a4e459a681.png)
![bat_60](https://user-images.githubusercontent.com/25363706/103580128-e8784d00-4ed9-11eb-9100-7c27d71b1347.png)
![bat_70](https://user-images.githubusercontent.com/25363706/103580129-e910e380-4ed9-11eb-9bfb-83610ec7006c.png)
![bat_80](https://user-images.githubusercontent.com/25363706/103580131-e910e380-4ed9-11eb-83fa-3436612b61c5.png)
![bat_90](https://user-images.githubusercontent.com/25363706/103580132-e9a97a00-4ed9-11eb-8c7c-d978681432c6.png)
![bat_max](https://user-images.githubusercontent.com/25363706/103580133-e9a97a00-4ed9-11eb-92cf-6153511c1835.png)
![bat_question](https://user-images.githubusercontent.com/25363706/103580134-e9a97a00-4ed9-11eb-9808-9bc06799a2a3.png)
![bat_verylow](https://user-images.githubusercontent.com/25363706/103580135-ea421080-4ed9-11eb-8e6a-dee6469e0702.png)
![bonus](https://user-images.githubusercontent.com/25363706/103580136-ea421080-4ed9-11eb-95ed-27db2804fc1b.png)
![signal_question](https://user-images.githubusercontent.com/25363706/103580137-ea421080-4ed9-11eb-8de8-412227fa81c0.png)
![signal-1-bar](https://user-images.githubusercontent.com/25363706/103580138-eadaa700-4ed9-11eb-9d2a-61c668d55d48.png)


Best

Uwe

**Reply by Gulivertx** [reply]
Nice thanks, but is it possible to have them in SVG format please ? Then I will try this icons and if the render is good in the app I will release them.

**Reply by uwewiest** [reply]
Hi Cedric,

forgot to mention I know SVG the ZIP file are the SVG items like it says it the name

Best

Uwe

**Reply by Gulivertx** [reply]
haha oups sorry
